lua/hbac/state.lua

```diff
@@ -1,16 +1,12 @@
 local M = {
+	pin_all = true,
 	pinned_buffers = {},
 	autoclose_enabled = false,
 }
 
 M.toggle_pin = function(bufnr)
-	if M.pinned_buffers[bufnr] == true then
-		M.pinned_buffers[bufnr] = false
-		return "unpinned"
-	end
-
-	M.pinned_buffers[bufnr] = true
-	return "pinned"
+	M.pinned_buffers[bufnr] = not M.pinned_buffers[bufnr]
+	return M.pinned_buffers[bufnr]
```

Comment:
This is part of the overall shift to letting the main `command` module handle notifications.